### ISSUE TITLE
BodyDetails() not populating material.name

### ISSUE THREAD

**Issue by Darkcyde13** [cmdrmcdonald/EliteDangerousDataProvider#147]
A question was raised on the Elite Forums about the Body Report not detailing the materials of a scanned body. After some testing, I think I found the problem to be BodyDetails() not populating the material.name variable for the list of available materials on the body. material.percentage seems to be populated ok though.

You can read the post [here](https://forums.frontier.co.uk/showthread.php/311614-EDDI-Scripts-and-EDDI-enabled-VA-Commands-Thread?p=5247326&viewfull=1#post5247326).

**Comment by cmdrmcdonald** [comment]
Looking at the code the variable name is 'material' so it would be material.material you would want to use.  Where are you seeing a reference to it being name?

**Comment by Darkcyde13** [comment]
{material.name} worked in the older version (v2.1.0) when I was just experimenting to test in the Body Scanned script.  I assumed it would also work in the latest version in the Body Report function.

I just tried {material.material} in both versions but neither of them say anything for this.  I used it in this line:
`{set allmaterials to cat(allmaterials, [material])} {material.material} {material.percentage}`
In v2.1.0 of Body Scanned, using {material.name} in this line speaks the name of the material, but in v2.2.0 of Body report it doesn't say anything. Likewise, {material.material} doesn't say anything in either of them.
